NoMansLand/Assets/Scripts/MovePhysics.cs

```diff
@@ -34,6 +33,14 @@ void Update()
         _userJumped = Input.GetButton("Jump");
     }
 
+    void OnCollisionEnter(Collision collision) 
+    {
+        if (collision.collider.tag == "Alien") 
+        {
+            Destroy(this.gameObject);
```

Comment:
Check out `CollectibleHit`. I do the same thing there. Is there any way you can perhaps merge the code in two (maybe rename the file to `PlayerCollision` to keep track of all player collisions and keep this separate from the Movement of the player?

Reply:
I included an if-statement within MovePhysics.cs to handle the collision between the Player and a Collectible. This renders CollectibleHit.cs unnecessary so I removed it.

Reply:
I suggest handling all player-related collisions here in this file. So if we want to handle more collisions that involve the player, we can just add if-cases to the current OnCollisionEnter function

Reply:
what if we want to add any more collisions? like collisions between two aliens? Would we ever need to deal with that?

Reply:
@gazijarin @sultansidhu  Feel free to correct me if I'm wrong